thrall/test/lib/ThrallStreamProcessorTest.scala

```diff
@@ -0,0 +1,49 @@
+package lib
```

Comment:
hmm, this is a [`git mv`](https://github.com/guardian/grid/pull/2929/commits/29014f6aa33bbac49ab53ccaaa62728979ac99b5) with edits, which apparently results in a delete and new file on GH?!

Reply:
It is heuristics based so depends how many edits in the file. If you want to be sure it's usually best to rename in one commit (which should be picked up as a mv) and then edits in a second commit.

Reply:
> If you want to be sure it's usually best to rename in one commit (which should be picked up as a mv) and then edits in a second commit.

I think I did this: the second commit on the branch is the move and the third is the edit 🤷‍♂️ .